benchmarks/deepcopy.cpp

```diff
@@ -56,7 +56,9 @@ void memcpy_1d(benchmark::State& state)
     for (auto _ : state) {
         std::memcpy(dst_data.data(), src_data.data(), dst_data.size() * sizeof(double));
     }
-    state.SetBytesProcessed(int64_t(state.iterations()) * int64_t(state.range(0) * sizeof(double)));
+    state.SetBytesProcessed(
+            static_cast<int64_t>(state.iterations())
+            * static_cast<int64_t>(state.range(0) * sizeof(double)));
```

Comment:
```suggestion
            static_cast<std::int64_t>(state.iterations())
            * static_cast<std::int64_t>(state.range(0) * sizeof(double)));
```

I think it is better to use `std::int64_t` for consistency